File: app/src/main/java/fr/paug/androidmakers/model/ScheduleSlot.java
package fr.paug.androidmakers.model;

public class ScheduleSlot {

    public final int room;
    public final int sessionId;
    public final long startDate;
    public final long endDate;

    public ScheduleSlot(int room, int sessionId, long startDate, long endDate) {
        this.room = room;
        this.sessionId = sessionId;
        this.startDate = startDate;
        this.endDate = endDate;
    }

    @Override
    public String toString() {
        return "ScheduleSlot{" +
                "room=" + room +
                ", sessionId=" + sessionId +
                ", startDate=" + startDate +
                ", endDate=" + endDate +
                '}';
    }

}

File: app/src/main/java/fr/paug/androidmakers/model/ScheduleSlot.kt
package fr.paug.androidmakers.model

class ScheduleSlot(val room: Int, val sessionId: Int, val startDate: Long, val endDate: Long) {
    override fun toString(): String {
        return "ScheduleSlot{" +
                "room=" + room +
                ", sessionId=" + sessionId +
                ", startDate=" + startDate +
                ", endDate=" + endDate +
                '}'
    }
}